test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 flake8; python_version >= '3.6'
 flake8-bugbear; python_version >= '3.6'
 flake8-pyi; python_version >= '3.6'
-pytest>=4.4.1; python_version >= '3.4'
+pytest==4.6.11
```

Comment:
Why are we pinning a specific version here?

Reply:
This was done by @srittau in this commit: https://github.com/python/typing/pull/866/commits/7a04d153bc24b4fc45646a45fd92d929b5077fb5

Reply:
Mostly, because pytest 4.6.11 is the last version supporting Python 2.7.